src/subapps/dataExplorer/PredicateSelector.tsx

```diff
@@ -91,9 +119,15 @@ export const PredicateSelector: React.FC<Props> = ({
 };
 
 export const DEFAULT_OPTION = '-';
-export const EMPTY_VALUE = 'Empty value';
+export const DOES_NOT_EXIST = 'Does not exist';
+export const EXISTS = 'Exists';
 export const CONTAINS = 'Contains';
-export type PredicateFilterT = typeof EMPTY_VALUE | typeof CONTAINS | null;
+
+export type PredicateFilterT =
+  | typeof DOES_NOT_EXIST
+  | typeof EXISTS
+  | typeof CONTAINS
+  | null;
```

Comment:
I suggest use this to be more `Typescripty`
`const PREDICATE_VALUES = [DEFAULT_OPTION,  DOES_NOT_EXIST, EXISTS, CONTAINS] as const`;
`type TPredicateValues = typeof PREDICATE_VALUES;`
and then 
`type TPredicateValue = TPredicateValues[0 | 1 | 2] or TPredicateValues[number]`
and then select by index when you need it `TPredicateValues[index]`

Reply:
bon idee! I'll do that in a new PR.